### PR TITLE
Bring in latest JsRuntimeHost with V8 inspector fixes

### PR DESCRIPTION
**Pull request by bghgary**
Also includes various testing fixes

### CODE REVIEW

File: Apps/UnitTests/CMakeLists.txt [hunk 1]
```diff
@@ -3,29 +3,30 @@ if(NOT((WIN32 AND NOT WINDOWS_STORE) OR (APPLE AND NOT IOS) OR (UNIX AND NOT AND
...
 
-set(NPM_SCRIPTS
+set(EXTERNAL_SCRIPTS
     "../node_modules/babylonjs/babylon.max.js"
     "../node_modules/babylonjs-materials/babylonjs.materials.js"
     "../node_modules/chai/chai.js"
     "../node_modules/mocha/mocha.js")
 
-set(HEADERS
-    "Shared/Tests.h")
```
Comment (PolygonalSun) [inline Apps/UnitTests/CMakeLists.txt:16]: Apple unit tests still have a reference to this file

Comment (PolygonalSun) [reply Apps/UnitTests/CMakeLists.txt:16]: Same with Ubuntu/Linux

**Review by PolygonalSun — COMMENTED** [review]
It also looks like the Install/Test/CmakeLists.txt will need changes to the `TestInstall` object

**Review by CedricGuillemet — APPROVED** [review]
LGTM

File: Apps/UnitTests/Shared/Shared.cpp [hunk 2]
```diff
@@ -11,41 +9,81 @@
...
         nativeCanvas.emplace(Babylon::Polyfills::Canvas::Initialize(env));
         Babylon::Plugins::NativeEngine::Initialize(env);
-        
-        env.Global().Set("SetExitCode", Napi::Function::New(env, [&exitCode](const Napi::CallbackInfo& info)
-        {
-            Napi::Env env = info.Env();
-            exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
-        }, "SetExitCode"));
+
+        auto setExitCodeCallback = Napi::Function::New(
```
Comment (ryantrem) [inline Apps/UnitTests/Shared/Shared.cpp:76]: Why add the variable?

Comment (bghgary) [reply Apps/UnitTests/Shared/Shared.cpp:76]: Formatting. Clang-format indents like crazy otherwise.

**Pull request merged**